template/nuxt/nuxt.config.js

```diff
@@ -108,7 +108,12 @@ module.exports = {
       }
     },<% } %><% if (ui === 'element-ui') { %>
     transpile: [/^element-ui/],
-    <% } %>
+    <% } %> <% if (ui === 'tailwind') { %>
```

Comment:
Space can be removed
```suggestion
<% } %><% if (ui === 'tailwind') { %>
```